service/src/main/java/uk/gov/mca/beacons/api/export/rest/BeaconExportRigUseDTO.java

```diff
@@ -36,4 +36,19 @@ public class BeaconExportRigUseDTO extends BeaconExportUseDTO {
 
   @Valid
   private String areaOfOperation;
+
+  @Valid
+  private String nsn;
+
+  @Valid
+  private String beaconPartNumber;
+
+  @Valid
+  private String pennantNumber;
+
+  @Valid
+  private String aircraftDescription;
```

Comment:
This seems a weird field to be on the rigUse?

Reply:
I don't know I didn't add that to be honest.. Do you want me to remove it?